docs/architecture/LOC_EXTRACTION_METHODOLOGY.md

```
@@ -0,0 +1,752 @@
+### Codebase Total
+
+```bash
+# Count ALL Python files in src/ (excluding tests)
+find src/image_preprocessing_detector -name "*.py" \
+  -not -path "*/tests/*" -type f -exec wc -l {} + | tail -1
+# Result: ~35,000 lines
+```
+
+### Workstream Sum
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)